lib/scope.js

```diff
@@ -178,7 +178,7 @@ module.exports = {
     *    the global permitted timeout */
 
     // Stop/resolve if too much time has passed
-    let expirationTime = 1.9 * 60 * 1000;  // m * s * ms
+    let expirationTime = scope.timeout - 200;  // ms
```

Comment:
Hmm... why subtract 200ms here? I can't figure it out.

Reply:
Good question. I'm actually not entirely sure. Weird. I remember thinking about this shortening again as I edited and deciding to it was needed, but I can't currently remember why. Maybe it was to make up for the time it took to get to this function? So that any error/warning message we put in here would be sure to get logged before the outer timeout triggered? If so, 200ms is a lot.

Reply:
#321